src/error/try_from_parsed.rs

```diff
@@ -15,6 +15,8 @@ pub enum TryFromParsed {
     InsufficientInformation,
     /// Some component contained an invalid value for the type.
     ComponentRange(error::ComponentRange),
+    /// Leap second is not allowed to occur at this date and time.
+    LeapSecondNotValid,
```

Comment:
Why not reuse the `ComponentRange` error type & variant? It can handle this quite easily.

Reply:
It feels a bit cryptic to fail second 60 as a plain range error when its validity actually depends on the content of other components.
The user might be interested to know why their input has failed if it was meant to be a leap second indeed. With an offset, it may not be obvious that the time is not actually at 23:59 UTC.

Reply:
There is a "conditional" field (or something with a similar name; I'm typing this off-hand) in the struct for things like this. A similar issue arises when validating the number of days in a month/year.